getting-started/aggregation.md

```diff
@@ -31,10 +31,10 @@ records that you need to look up to perform your query.
 
 <CaggsTypes />
 
-In this section, you create a simple aggregation by finding the average trade
-price over a week. Then, you create a materialized
-view, transform it into a continuous aggregate, and query it for more
-information about the trading data.
+In this section, you create a materialized view, transform it into a continuous
```

Comment:
I know the text mentioned it before, and it's sad I just realize it now, but you do not create a materialized view and transform it into a continuous aggregate. You create a continuous aggregate through a similar syntax to a materialized view in PostgreSQL :-)